hammerdb/hammerdb_scripts/mssql/build_mssql_tpcc.sh

```
@@ -0,0 +1,81 @@
+## Collecting system information
+## Setting up variables for the run and Collecting system information ####
+Usercount="50"
+whc="500"
+benchmark_name="Hammerdb-tpcc"
+echo ${benchmark_name} > ${benchmark_run_dir}/user-benchmark-name.txt
```

Shellcheck (0.11.0)

[warning] benchmark_run_dir is referenced but not assigned.

(SC2154)